### DISCUSSION TITLE
Separate Artists with Same Name?

### DISCUSSION THREAD

**Discussion by zaptrem** [beetbox/beets#4672] [Q&A]
Let's say there are two Chris Browns. One is the one you're thinking of, one is completely unrelated and I'd like to rename him. Now scale that to thousands of artists. How do I rename less popular artists when their names conflict with those of bigger artists? Ideally, it would rename all the songs by the less popular artist to "Chris Brown 2" or something.

**Comment by sampsyo** [comment]
Good question!

This is more background than you wanted, but dealing with this in The Right Way™ would entail a change to the schema of beets. A fun fact is that in the early days of beets, we had the same problem with identically-named albums (e.g., Weezer has several self-titled albums). That's why beets has an explicit representation of an album that's not just implied by the metadata. We would need to do something similar to create an explicit schema representation for artists too.

Anyway, barring that, a good option at this point would be to create your own custom field to disambiguate them. That is, do `beet modify <something> artistdisambig='the other one'`, for instance, and then use `$artistdisambig` in your path formats to sort the two different Chris Browns into two different directories.
